src/frequenz/client/microgrid/__init__.py

```diff
@@ -41,14 +60,31 @@
     "ComponentMetricId",
     "ComponentType",
     "Connection",
+    "DataLoss",
     "EVChargerCableState",
```

Comment:
The list of symbols in `__all__` is much more readable from the jumbled import statements where the symbols are not aligned and are grouped by the file they come from and not the category.  Please don't remove it.

Reply:
> this is a separate conversation, so let's not have it in this PR, I prefer to focus on this PR here

I didn't start it 😈 

> For me __all__ is just noise

and you didn't really stop either. 😠 

> if it really adds clarity for you, I can add it

I like doing it, but I mentioned optional anyway, so it is up to you.

Reply:
One more thing while I think about this (I was looking into it to see if I would use a different categorization or the same as in files). If these comments are really intended to the users, I think we should add that classification in the module documentation instead, if we do that, we have some duplication, we still need to more or less list all the symbols twice, but I do see much more added value there, if this classification is shown in the docs.

In any case, this this is pre-existing code and not something added by this PR, I would rather do this as a separate PR, when improving the docs.

Reply:
> and you didn't really stop either. 😠

Haha, I meant just about removing `__all__` completely.

> if it really adds clarity for you, I can add it

I know see the value in the grouping as user documentation, but I don't want to get into writing the docs now that the project will change a lot when the upgrade to v0.17.0 happens.

If you think the current mapping of files/categories makes sense, I could add comments to match the files, that's not a problem as it is very low effort. Just let me know and I will add it, is just a few tab-tab with copilot.

If not, I will need to spend some time thinking which categories are appropriate, and if I do that, I will want to also map those categories to files as it should be. This will be something much more time consuming and again kind of pointless just before the upgrade to v0.17.0.